### PR TITLE
Modify Rule S2681: LaYC comment blocks

### PR DESCRIPTION
**Pull request by frederic-tingaud-sonarsource**

### CODE REVIEW

File: rules/S2681/csharp/example.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+if(condition) FirstActionInBlock();  // Noncompliant
+  SecondAction();  // Executed unconditionally
+----
+
+[source,csharp]
+----
+string str = null;
+for (int i = 0; i < array.Length; i++) 
+  str = array[i];
+  DoTheThing(str);  // Noncompliant; executed only on last array element
```
[the remark this answers is not in the document]
Comment (frederic-tingaud-sonarsource) [reply rules/S2681/csharp/example.adoc]: Not a fan of this one. The comment is already at the end of a long line, I think the loss in readability from a long comment outweighs the gain from having complete sentences.

File: rules/S2681/common/example.adoc [hunk 2]
```diff
@@ -1,19 +1,29 @@
...
   secondAction();  // Executed unconditionally
+----
 
+[source,java]
+----
 if (condition); secondAction();  // Noncompliant; secondAction executed unconditionally
+----
 
+[source,java]
+----
```
[the remark this answers is not in the document]
Comment (frederic-tingaud-sonarsource) [reply rules/S2681/common/example.adoc:26]: simplified to remove the variable declaration

File: rules/S2681/csharp/example.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (amelie-renard-sonarsource) [reply rules/S2681/csharp/example.adoc]: I see your point. However I'd still add "the": "Noncompliant: executed only on the last array element"

**Review by amelie-renard-sonarsource — APPROVED** [review]
See my answer to your comment. Otherwise, all good!

File: rules/S2681/csharp/example.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (frederic-tingaud-sonarsource) [reply rules/S2681/csharp/example.adoc]: Yes, I changed it to `// Noncompliant: executed only on the last element`

**Pull request merged**